libs/wingsdk/src/ex/react-website.ts

```diff
@@ -0,0 +1,170 @@
+import { existsSync } from "fs";
+import { isAbsolute, join, resolve } from "path";
+import { Construct } from "constructs";
+import { BaseWebsiteProps, Website } from "../cloud/website";
+import { fqnForType } from "../constants";
+import { App } from "../core";
+import { Resource, Node } from "../std";
+
+const DEFAULT_BUILD_FOLDER = "/build";
+const DEFAULT_START_COMMAND = "npm run start";
+const DEFAULT_BUILD_COMMAND = "npm run build";
+const DEFAULT_PORT = 3001;
+
+export const WING_JS = "wing.js";
+
+/**
+ * Global identifier for `ReactWebsite`.
+ */
+export const REACT_WEBSITE_FQN = fqnForType("ex.ReactWebsite");
+
+/**
+ * Options for `ReactWebsite`.
+ */
+export interface ReactWebsiteProps {
+  /**
+   * The path to the React app root folder- can be absolute or relative to the wing folder
+   */
+  readonly projectPath: string;
+  /**
+   * The path to the React app build folder- relative to the `projectPath`
+   * @default "/build"
+   */
+  readonly buildFolder?: string;
+  /**
+   * A command for starting React app locally
+   * @default "npm run start"
+   */
+  readonly startCommand?: string;
+  /**
+   * A command for building the React app
+   * @default "npm run build"
+   */
+  readonly buildCommand?: string;
+  /**
+   * In sim, if `true` - will use the start command, and if `false` - the build command
+   * @default false
+   */
+  readonly isDevRun?: boolean;
```

Comment:
I wanted to enable two possible behaviors on the sim target (similar to when developing a react app)-
1- The developer is working on his react app- interacting with his files and developing more features (98% of the time I guess)
2- The developer is done working on his app, so he builds it (and its servers over a nodeJs or a similar static server ) - I usually did it only when testing specific parts in that area, it'll used by the preview env (otherwise won't be accessible)
As a developer, I'd still want to be able to choose whether to run build or start- but I agree that the default of the sim env should be start

Reply:
that's a good question, `ENV` or `NODE_ENV` will be set to "production" when the app is in the cloud (I guess we don't want to set it that way locally since we can accidentally use production apis and services... )
it's set by the tool that builds the react to its dist files 
I think I just remove that part to eliminate confusion, good catch Chris!